spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/BravePropagationConfigurations.java

```diff
@@ -101,8 +101,18 @@ BaggagePropagation.FactoryBuilder propagationFactoryBuilder(
 		private Factory createThrowAwayFactory() {
 			return new Factory() {
 
+				/**
+				 * @deprecated end users and instrumentation should never call this, and instead
+				 * use {@link #get()}. This only remains to avoid rev-lock upgrading to Brave 6.
+				 */
+				@Deprecated
```

Comment:
There's no point in deprecating this as it's package-private API for which the javadoc is not published.